networkit/test/test_clique.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import unittest
+import os
+import networkit as nk
+import numpy as np
+import scipy
+
+class Test_Cliques(unittest.TestCase):
```

Comment:
To be consistent with other Python tests:
```suggestion
class TestCliques(unittest.TestCase):
```